src/dataStructure/DoublyLinkedList.tsx

```diff
@@ -0,0 +1,62 @@
+class Node<T> {
+  value: T;
+  previous: Node<T> | null;
+  next: Node<T> | null;
+
+  constructor(value: T) {
+    this.value = value;
+    this.previous = null;
+    this.next = null;
+  }
+}
+
+class DoublyLinkedList<T> {
```

Comment:
This object is really good stuff, but not sure if should be something to use inside the stepper but not REQUIRED to make it work.